src/admin/testPages.tsx

```diff
@@ -13,13 +13,25 @@ import * as querystring from 'querystring'
 import * as _ from 'lodash'
 import * as url from 'url'
 
+const IS_LIVE = NODE_BASE_URL === "https://owid.cloud"
```

Comment:
@mispy Not sure if this is a good way to check whether this is the live environment—let me know if there's a better one!

Reply:
That's the way I did it for the little indicator in the navbar, yep. There's probably something more systematic but it works for now.